cfme/tests/infrastructure/test_host.py

```diff
@@ -312,3 +312,41 @@ def _finalize():
             actual_state, power_state
         ),
     )
+
+
+@test_requirements.general_ui
```

Comment:
I believe it should be `infra_hosts` FA. General UI is a different FA and has a different owner.